src/js/components/Meter/Circle.js

```diff
@@ -145,6 +159,8 @@ const Circle = forwardRef((props, ref) => {
         strokeWidth={height}
         strokeLinecap={round ? 'round' : 'square'}
         fill="none"
+        strokeDasharray={type === 'semicircle' ? circumference : 0}
```

Comment:
I don't think we want this. We should use `arcCommands` to add a path in the semicircle case, instead of using `<circle />`.

Reply:
> Math ideas for semicircle:
> 
> * start angle should be 270, end angle should be 90
> * center Y should be `width` instead of `width / 2`
> * `anglePer` should be `180 / max`
> * we should use something like `angle % 360` instead of `Math.max(360, angle)` in `arcCommands()`, just let the angle loop around
> * resulting viewBox should have `height = width / 2`
> * we'll have to draw an arc instead of using `<circle />` for the background track
> 
> Hopefully this gives some ideas on things to work from.

So of course I saw this after I pushed up some changes! I think I had thought of it a little different but I agree with the
> * start angle should be 270, end angle should be 90
> * center Y should be `width` instead of `width / 2`
> * `anglePer` should be `180 / max`
> * resulting viewBox should have `height = width / 2`

I think the 2 things I would like to discuss further today are:   
> * we'll have to draw an arc instead of using `<circle />` for the background track
> * we should use something like `angle % 360` instead of `Math.max(360, angle)` in `arcCommands()`, just let the angle loop around

Just to set up our discussion 👆🏻